### DISCUSSION TITLE
DLL load failed while importing create_mesh_surf

### DISCUSSION THREAD

**Discussion by coder-fny** [simnibs/simnibs#310] [2. General]
It looks like there are some DLL libraries missing when importing the .pyd file, I checked the missing DLLs with dependency walker, downloaded some and put them in the cgal directory, but the pumutilities.dll is still missing, but I don't know where to get this file, and I don't know if this is the reason why the program can't run

**Comment by Kstoffer** [comment]
Can you say something about how you installed simNIBS and which version it is. Typically the dll's should be linked automatically during the cython compilation procedure (run via the setup.py script) or automatically be installed in the correct locations if you are using the packaged installer.

**Reply by coder-fny** [reply]
Thanks for the reply, I've solved it, I reinstall it
